lib/iris/fileformats/grib/__init__.py

```diff
@@ -962,7 +970,7 @@ def as_pairs(cube):
 def as_messages(cube):
     """
     Convert one or more cubes to GRIB messages.
-    Returns an iterable of grib_api GRIB messages.
+    Returns an iterable of grib_api GRIB message IDs.
```

Comment:
I think 'handles' would be lot clearer than 'IDs', as Python objects have 'ids'.  But then I see that 'id' is used in the grib_api documentation.

This applies to all the last 3 low-level routines : as_pairs / as_messages / save_messages.
At least this one is explicit in referring to **_"grib_api_** GRIB message IDs"
I can't create a comment for it, but I think the line 950 docstring should be improved to a similar style.